src/routes/AllTopics/AllTopics.js

```diff
@@ -99,7 +99,7 @@ export default class AllTopics extends Component {
     } = this.props
     const { totalTopicsCached } = this.state
 
-    return <FullPageModal fullWidth>
+    return <FullPageModal fullWidth goToOnClose={baseUrl({ ...routeParams, view: undefined })}>
```

Comment:
That looks like a really good solution to that wrinkle 👍

Reply:
Heh, it's funny cos I cherry-picked this from my old branch and we didn't like it there but somewhere along the line our relationship to the navigation logic seems to have evolved (;